files/default/sql/ddl/updates/0.9.0.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE jupyter_project ADD CONSTRAINT unique_hdfs_user UNIQUE (hdfs_user_id);
+
+ALTER TABLE jupyter_project DROP FOREIGN KEY `FK_103_525`;
```

Comment:
You cannot assume the name of the constraint. You have to query the information schema to get the correct name by querying the `information_schema` db

Reply:
Added the sql for querying, migration and rollback both work